apps/pie-microsite/src/content/pages/designers/getting-started/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Getting started
+    order: 1
+---
+
+## Introduction 
+
+[Figma](https://www.figma.com/ui-design-tool) is PIE’s design tool of choice for building our design system. All design-related foundations, components and assets are stored within Figma libraries and files to ensure the whole digital ecosystem can be accessed by any team.
+
+You can access Figma through the browser, but it is recommended that you use the desktop app. If Figma’s desktop app is not installed on your computer, get in touch with your manager to request this.
+
+As a designer within the Just Eat Takeaway organisation, you’ll automatically have access to all of PIE’s libraries and files. However, if that’s not the case, please [get in touch with the PIE team]() to request access.
+
+{% notification {
+type: "information",
+title: "New to Figma?",
+message: "If you’re new to Figma, check [these tutorials](https://www.figma.com/resources/learn-design/?fuid=1093444461414143879) that will help you learn how to use the tool in no time."
+} %}
+
+---
+
+## How is PIE structured?
+
+We have divided our foundations, components and assets into multiple libraries, so they can be easily found and referenced. All of them can be found on the [PIE Team]() section in Figma.
+
+You’ll need to enable these libraries if you want to use the PIE design system for your products. If you don’t know how to do that, please read our [‘How to enable libraries in Figma’]() guide.
+
+
+{% contentPageImage {
+src: "../../../../../assets/img/designers/getting-started/libraries-wheel.svg",
+mobileSrc: "../../../../../assets/img/designers/getting-started/libraries-wheel_narrow.svg",
+width: "512px",
+caption: "A map of how Libraries are structured in Figma."
+} %}
+
+
+### Foundations libraries
+
+These libraries contain the building blocks for our design system. In these libraries you’ll find the styles created using colour, typography, elevation, radius and spacing (among others). These foundations should remain consistent across components, products and pillars.
+
+PIE designers own and maintain our foundation libraries, which are split into two different themes (PIE 2.0 Light and PIE 2.0 Dark). Make sure you’re working from the right library, and check with the PIE team if you’re unsure about which one you should use.
+
+### Core Components libraries
+
+Our core component libraries are made up of all the components that are used by multiple business areas and teams. These include components like buttons, labels, chips, input fields (among others). These components should remain consistent across products and pillars.
+
+PIE designers own and maintain our components libraries, which are split into two different themes (PIE 2.0 Light and PIE 2.0 Dark). Make sure you’re working from the right library, and check with the PIE team if you’re unsure about which one you should use.
```

Comment:
It's funny that we repeat the same thing here as we do on line 43 in the foundation libraries section. Maybe we should vary the language slightly? Or add something like, `similarly to foundation libraries, these components are maintained and owned by PIE designers`. Super nit, but avoids people from thinking they've read the same bit twice (aka me 😄 )

Reply:
thanks @jordifornellsJET 😄